js/components/discussions/thread-form.vue

```diff
@@ -24,24 +22,32 @@ export default {
       return {
           // Flags to `true` when the form is in sending process. This prevents
           // duplicate POSTing.
-          sending: false
+          sending: false,
+          comment: ''
       }
   },
   methods: {
+      /**
+       * Prefill the form and focus the comment area
+       */
+      prefill(comment) {
```

Comment:
These are not the same forms and there were already quite similar.
I can but it will be overly complex for nothing (a mixin or an intermediate component for both).